src/main.ts

```diff
@@ -1,11 +1,30 @@
+import { Logger } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { NestFactory } from '@nestjs/core';
+import { NestExpressApplication } from '@nestjs/platform-express';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { join } from 'path';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
-  const configService = app.get(ConfigService);
-  const port = configService.get('PORT');
-  await app.listen(port);
+  const app = await NestFactory.create<NestExpressApplication>(AppModule);
+  //app.useStaticAssets(join(__dirname, '..', 'views'))
+  const globalPrefix = 'api';
+  app.setGlobalPrefix(globalPrefix);
+  const port = process.env.PORT || 3333;
```

Comment:
Not a critical issue, the more NestJS way of doing this will be
```suggestion
  const configService = app.get(ConfigService);  
  const port = configService.get('PORT');
```